src/pages/images/ImageSelector.tsx

```diff
@@ -210,19 +210,20 @@ const ImageSelector: FC<Props> = ({ onSelect, onClose }) => {
           : item.variant;
 
       const getSource = () => {
-        if (item.created_at) {
-          return "Local";
+        let source = "Custom";
+        if (!item.cached && item.created_at) {
+          source = "Local";
         }
         if (item.server === canonicalServer) {
-          return "Ubuntu";
+          source = "Ubuntu";
         }
         if (item.server === minimalServer) {
-          return "Ubuntu Minimal";
+          source = "Ubuntu Minimal";
         }
         if (item.server === imagesLxdServer) {
-          return "LXD Images";
+          source = "LXD Images";
         }
-        return "Custom";
+        return source + (item.cached ? " (cached)" : "");
```

Comment:
I think we can better present this as a small muted text to not draw too much attention through the two lines with brackets

![image](https://github.com/user-attachments/assets/bc8cc015-5bde-4fc7-8eda-d9d185702775)

Reply:
great idea, done! :+1: